inspirehep/modules/workflows/tasks/refextract.py

```diff
@@ -28,6 +28,7 @@
 
 from refextract import extract_journal_reference, extract_references_from_file
 
+from inspirehep.config import REFEXTRACT_JOURNAL_KB_PATH
```

Comment:
oh ok, I didn't know that! And also I tested the query with all the journals from here http://inspirehep.net/dumps/inspire-dump.html and it takes around 0,33 sec.